altos-rust/altos-core/libs/heap/free_list_allocator/src/free_list.rs

```diff
@@ -141,7 +143,7 @@ impl FreeList {
         // Adjust the heap size down based on alignment change to starting position
         // and then adjust it down again if it's not aligned to block header size.
         let use_heap_size =
-            alignment::align_down(heap_size - align_diff, mem::size_of::<BlockHeader>());
+        alignment::align_down(heap_size - align_diff, mem::size_of::<BlockHeader>());
```

Comment:
Why was this changed? I feel like indenting this makes it more clear that it's not a separate statement, but part of the assignment. The style guide doesn't seem to specify how this should be done.